src/components/AppHeader/index.css

```diff
@@ -0,0 +1,26 @@
+.app-header {
+  display: flex;
+  min-height: 45px;
+}
+
+.app-header__logo {
+  margin-right: auto;
+}
+
+.login {
```

Comment:
The AppHeader component should follow the BEM structure with classes starting from .app-header__login; otherwise, the login should be a separate component.